### PR TITLE
GEOMESA-1289 Download script for tdrive doesn't work

### PR DESCRIPTION
**Pull request by atallahhezbor**
Updates URLs for TDrive and handles scraping complaints

Signed-off-by: Atallah Hezbor <atallahhezbor@gmail.com>

### CODE REVIEW

File: geomesa-tools/bin/download-data.sh [hunk 1]
```diff
@@ -53,14 +53,16 @@ case "$type" in
   tdrive)
 
     echo "Note: each zip file contains approximately one million points"
-    read -p "Download how many zip files? " NUM
+    read -p "Download how many zip files? (9 total) " NUM
```
Comment (jnh5y) [inline geomesa-tools/bin/download-data.sh]: Are there only 9 files now?  I previously downloaded like ~4800....

Comment (atallahhezbor) [inline geomesa-tools/bin/download-data.sh]: They may have grouped some them together, but i don't feel like their claim of 15 million points still holds.
[See here](https://www.microsoft.com/en-us/research/publication/t-drive-trajectory-data-sample/)

Comment (elahrvivaz) [inline geomesa-tools/bin/download-data.sh]: When I downloaded it a few months ago, there were zips 01-014. But is seems that 01-05 are no longer hosted...
That page doesn't render fully for me in chrome, either.

**Comment by elahrvivaz** [comment]
actually the links to the first 5 still work: https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/02.zip

**Comment by atallahhezbor** [comment]
Should we count on those remaining hosted and always download them? Worst case the user gets what will look like a bad zip

**Comment by atallahhezbor** [comment]
Or we could check the length of the file first

**Comment by elahrvivaz** [comment]
@atallahhezbor I pushed a change that works for me - can you try it out?

**Comment by elahrvivaz** [comment]
actually doing a length or md5 check on the files is a pretty good idea - there are only 14 of them so we could compare them in the script

**Comment by elahrvivaz** [comment]
actually not sure if you can access it but wget shows the response type - we could just check that it's application/zip.

**Comment by elahrvivaz** [comment]
oops, i pushed to the wrong branch - here's the change I made: https://github.com/ccri/geomesa/commit/c201bbb5a46381e50198ce463a011b3afc4b2b1d

**Comment by atallahhezbor** [comment]
That worked for me! Not sure why I was getting 403s with my user agent.
For checking type, we could grep the output of wget --spider?

**Comment by jnh5y** [comment]
Merged as 034fcad6230d3db834832316b64d616f7847cd56.  Thanks @atallahhezbor @elahrvivaz!

**Pull request closed without merging**